src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -82,8 +86,7 @@ public class DriveTrain extends SubsystemBase {
       new DifferentialDriveKinematics(Robot.getDriveTrainConstant("TRACK_WIDTH").asDouble());
 
   // Defines the odometry of the drive train, which is used to calculate the position of the robot.
-  private final DifferentialDriveOdometry odometry;
-
+  private final DifferentialDrivePoseEstimator poseEstimator;
```

Comment:
I'd like to have a way to fallback to the non-photonVision odometry stuff.  Ideally, it would be a runtime config option (e.g. enable/disable photonVision buttons on shuffleboard), but compile time is a reasonable second option.

Reply:
The pose estimator also works without photonvision, and the PoseEstimator class extends odometry, so it has the same features. I have already tested it with photonvision disconnected, and it does fallback onto the encoders and navx.

Reply:
@ParkerMeyers, how do we disable photonVision at run-time in case it is not behaving as expected?